plugins/blockvault_client/test/netlayer/CMakeLists.txt

```diff
@@ -8,36 +8,25 @@ include_directories(${Boost_DIR}/../../../include)
 link_directories(${OPENSSL_ROOT_DIR}/lib)
 #find_library(SSL_LIB libssl.a REQUIRED)
 #find_library(CRYPTO_LIB libcrypto.a REQUIRED)
-find_library(COROUTINE_LIB libboost_coroutine.a REQUIRED)
-find_library(THREAD_LIB libboost_thread-mt.a REQUIRED)
-find_library(CONTEXT_LIB libboost_context-mt.a REQUIRED)
+#find_library(COROUTINE_LIB libboost_coroutine.a REQUIRED)
+#find_library(THREAD_LIB libboost_thread-mt.a REQUIRED)
+#find_library(CONTEXT_LIB libboost_context-mt.a REQUIRED)
```

Comment:
Commenting out isn't the cleanest approach (this, along with the other commented out CMake logic).

Reply:
upper comment make sense, if you have any more thinking is also welcomed to put here.